scss/mixins/_forms.scss

```diff
@@ -1,55 +1,51 @@
-//DEPRECATE 1.6
-
 // Form Mixins
 @mixin fd-form-base {
     @include fd-reset();
     @include fd-type("0");
     appearance: none;
     -webkit-appearance: textfield;
     -moz-appearance: textfield;
-    font-size: inherit;
```

Comment:
This is non self contained styles friendly - fd-reset() takes care of setting the font-size to 14px.